### ISSUE TITLE
Vue support

### ISSUE THREAD

**Issue by saadq** [saadq/lynt#2]
Would be nice to have a `--vue` flag. This shouldn't be problematic on the ESLint side of things due to [eslint-plugin-vue](https://github.com/vuejs/eslint-plugin-vue). Unfortunately, I'm not sure if this can be handled on the TSLint side of things though due to single-file components. More info about that here: https://github.com/palantir/tslint/issues/2099

Due to my limited (basically zero) knowledge of Vue, I could definitely use help in this department.

**Comment by g-plane** [comment]
I think we can support it with ESLint first. This will make the linter available for those who uses pure JavaScript and Vue.js.

**Comment by saadq** [comment]
Yep, I think I'll go with that and just add a disclaimer in the README about using `--vue` and `--typescript` together.

Note to self: 

* https://github.com/mysticatea/vue-eslint-parser
* https://github.com/vuejs/eslint-plugin-vue
